src/components/composer/composer/composer.ts

```diff
@@ -329,9 +329,10 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.env.model.dispatch("STOP_EDITION", { cancel: true });
   }
 
-  private processF4Key() {
+  private processF4Key(ev: KeyboardEvent) {
     this.env.model.dispatch("CYCLE_EDITION_REFERENCES");
     this.processContent();
+    ev.stopPropagation();
```

Comment:
Whenever it's possible, I prefer to have functions like `stopPropagation`, `preventDefault`, ... in the beginning of the function. But it's a personal preference, feel free to ignore